cross/emacs/PLIST

```diff
@@ -0,0 +1 @@
+bin:bin/emacs
```

Comment:
You should include here all files required to be packaged with proper "type": https://github.com/SynoCommunity/spksrc/wiki/PLIST-files
So "lnk" has to be used for `bin/emacs`

Reply:
I will give that a try. Any ideas on the `LD_LIBRARY_PATH` problem?